c31-bywheat/by-wheat-nutshell-react
# Cm registration

# Description

Added register button on login form. When clicked redirects to register page allowing you to fill out name email and password fields. Register button then saves the new user and redirects you to the login page allowing you to log in.

Applies to issue # 6

## Type of change

Please delete options that are not relevant.




# Testing Instructions

After starting up application click register button. Fill out all fields then click register it should save this data then allow you to log in.



# Checklist:
